VehicleMsg cur_vehicle_state
CurrentLaneMsg current_lane
PerpendicularLaneMsg[] next_intersection
ParallelLaneMsg[] adjacent_lanes
float64 speed_limit
RewardInfoMsg reward_info
PedestrainMsg[] pedestrain
GlobalPathMsg global_path 